027-FHIRPoweredHealthcare/readme.md

```diff
@@ -1,78 +1,69 @@
 # FHIR Powered Healthcare
 ## Introduction
-Contoso Healthcare is implementing a FHIR (Fast Healthcare Interoperability Resources) powered event-driven serverless platform to ingest and transform patient data and medical events from EHR (Electronic Health Record) systems into a HL7 FHIR standard format and persist them to a centralized FHIR Compliant store.  To support post-processing of medical events, an event is generated whenever FHIR CUD (create, update and delete) operations occurs in FHIR Server.  The platform will enable exploration of FHIR patient data through frontend web apps and SMART on FHIR apps.
+Contoso Healthcare is implementing FHIR (Fast Healthcare Interoperability Resources) to rapidly and securely exchange data in the HL7 FHIR standard format with a single, simplified data management solution for protected health information (PHI). Azure API for FHIR is fully managed, enterprise-grade FHIR service in the cloud lets you quickly connect existing data sources, such as electronic health record systems and research databases. Create new opportunities with analytics, machine learning, and actionable intelligence across your health data.
 
-You will implement a collection of FHIR reference architectures in the Microsoft **[Health Architectures](https://github.com/microsoft/health-architectures)** for FHIR use cases that best fit Contoso Healthcare requirements. Below is the holistic conceptual end-to-end Microsoft Health architectures for Azure API for FHIR.
+You will implement a collection of FHIR reference architectures frome **[Microsoft Health Architectures](https://github.com/microsoft/health-architectures)** that best fit Contoso Healthcare requirements. Below is the holistic conceptual end-to-end Microsoft Health architectures for Azure API for FHIR.
 ![Health Architecture](./images/HealthArchitecture.png)
 
 ## Learning Objectives
```

Comment:
The "Learning Objectives" section on the root readme.md reads as a list of every service in the Azure catalog! :)  Not all of these services are learning objectives for this hack.  I would consider reducing this list to just the FHIR related services, and if you want to list other Azure services, consider moving them to the "Learning Resources" sections of one of the challenges?

Reply:
These updates have been merged to our master branch and ready for PR.  Let us know if there is anything else.

Reply:
Thanks Richard!